static/js/reviewForm.js

```diff
@@ -0,0 +1,76 @@
+export default function handleReviewForm() {
+
+    const toggle = document.querySelector("#toggle-review-form");
+    toggle.addEventListener("click", () => {
+        showModal();
+    });
+
+    document.body.addEventListener("click", e => { //Hide modal when clicking outside of review-form
+        if (e.target.classList.contains("modal-container") && !e.target.classList.contains("toggle-review-form")) {
+            hideModal();
+        }
+    });
+
+    const form = document.querySelector(".review-form");
+    form.addEventListener("submit", async e => {
+        e.preventDefault();
+
+        //POST form to server using fetch here
+        const data = new URLSearchParams(new FormData(form));
+        const currentPath = window.location.pathname;
+        const res = await fetch("/api" + currentPath  + "/reviews", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/x-www-form-urlencoded",  
+            },
```

Comment:
This is not _wrong_, but it's a bit uncommon to use URL-encoded form data for APIs nowadays, and more common to just use JSON.